CHANGELOG.md

```diff
@@ -22,6 +22,13 @@ IMPROVEMENTS:
   * `consul.hashicorp.com/transparent-proxy-exclude-outbound-cidrs` - Comma-separated list of IPs or CIDRs to exclude.
   * `consul.hashicorp.com/transparent-proxy-exclude-uids` - Comma-separated list of Linux user IDs to exclude.
 
+* Connect: Add the ability to set default tproxy mode at namespace level via label. [[GH-501](https://github.com/hashicorp/consul-k8s/pull/510)]
+
+  Setting `consul.hashicorp.com/transparent-proxy` to `true/false` will have the following behaviour:
+  * If set as a `pod annotation` - This will define whether tproxy is enabled/disabled for the pod.
+  * If set as a `namespace label` - This will define the default behaviour for pods in this namespace which do not have their respective annotation set.
+  * If not set on either pod or namespace - The default behaviour will be defined by the `connectInject.transparentProxy.defaultEnabled` helm value.
```

Comment:
Given this a change in `consul-k8s`, can you rephrase the `connectInject.transparentProxy.defaultEnabled` helm value as the equivalent flag and sub-command it gets set on in `consul-k8s`?
Additionally, I'd phrase it as annotation on the Pod and label on the Namespace. I don't think `pod annotation` and `namespace label` seem accurate. 
Conversely:
Setting the annotations `consul.hashicorp.com/transparent-proxy` to `true/false` on the pod will define whether tproxy is enabled/disabled for the pod. Maybe?